docs/asciidoc/static/managing-multiline-events.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[multiline]]
+=== Managing Multiline Events
+
+Several use cases generate events that span multiple lines of text. In order to correctly handle these multline events, 
+Logstash needs to know how to tell which lines are part of a single event.
+
+Whenever possible, avoid using multiline processing at all. When multiline processing is required, implement the
```

Comment:
"avoid using" - if we make this recommendation, we should say why, and right after saying to avoid it. Thoughts?